science/ncbi_tools/Portfile

```diff
@@ -56,9 +110,12 @@ destroot {
     foreach manpage ${manpages} {
         xinstall -m 444 ${worksrcpath}/ncbi/doc/man/${manpage} ${destroot}${prefix}/share/man/man1/
     }
+    xinstall -m 755 -d  ${destroot}${prefix}/data
+    foreach datafile ${datafiles} {
+        xinstall -m 444 ${worksrcpath}/ncbi/data/${datafile} ${destroot}${prefix}/data/
+    }
```

Comment:
Note that you don't need a `foreach` loop here. It could be just:
```
xinstall -m 444 -W ${worksrcpath}/ncbi/data/ {*}${datafiles} ${destroot}${prefix}/data/
```

Reply:
Good to know! I was replicating the style of the other file operations the were already in the portfile -- your suggestion is cleaner for sure.